### ISSUE TITLE
Scaladoc external references jdk11

### ISSUE THREAD

**Issue by ashawley** [scala/bug#10675]
It's no longer possible for scaladoc to reference the Java class library from Java 9, now, which was discovered here https://github.com/ThoughtWorksInc/sbt-api-mappings/issues/8 by the Akka team.  The Akka docs have a few Java refs in their api docs, e.g. https://doc.akka.io/api/akka/current/akka/japi/Util$.html

Presumably, the Java-9 `rt.jar` hack would work from https://github.com/scala/scala-dev/issues/139, but in the long-term scaladoc will need `module-path` support and a way to point to modules  like `java.base`...

```
scala> java.lang.ModuleLayer.boot.configuration.modules.stream.filter(
         m => m.reference.descriptor.name == "java.base"
       ).forEach(
          m => println(m.reference)
       )
[module java.base, location=jrt:/java.base]
```

**Comment by sjrd** [comment]
Here is a workaround that we've been using in the Scala.js build for a while: https://github.com/scala-js/scala-js/blob/aede3c83fc6ab9132f475b98a662d0aaaa77dfb8/project/Build.scala#L233-L237

**Comment by ashawley** [comment]
Ok, good to know.  Thank you.   I gave your hack a go in ThoughtWorksInc/sbt-api-mappings#14.

**Comment by ashawley** [comment]
@sjrd Looks like it works for 2.12, but not 2.11.13 for some reason.  Your experience as well?

**Comment by sjrd** [comment]
Well we have https://github.com/scala-js/scala-js/issues/3152, but otherwise it seemed like it was working fine (for other projects in our build, beside `library`). I'm not sure that I actually carefully checked the output, though. I am only certain that it builds.

**Comment by ashawley** [comment]
Ok, I tried looking through the Jenkins builds for scalajs, but I don't think Java 9 is in the matrix.  I'll try the library project locally on Java 9....

Yeah, doesn't look like the hack works on 2.11

https://travis-ci.org/ashawley/scaladoc-test/jobs/335874903

**Comment by ashawley** [comment]
This probably won't be fixed, but worked in 2.12.

It doesn't work again in 2.13, but that can be superseded by #11839.
